docker/local-universe/README.md

```diff
@@ -190,3 +190,13 @@ For example, if you want dev-universe, then add `--server_url http://dev-univers
   ]
 }
 ```
+
+## Making changes...
+
+###  ...to the `universe-static` image
+
+1. Update `Dockerfile.static` with the changes
+1. Bump the default value of `static_version` in `Makefile`
+1. Create a pull request with the above changes
+1. Once it is merged, run the [Publish universe-static image](https://teamcity.mesosphere.io/viewType.html?buildTypeId=Oss_Universe_2_PublishUniverseStaticImage&branch_Oss_Universe_2=%3Cdefault%3E&tab=buildTypeStatusDiv) build, providing the new version number when prompted
```

Comment:
I am concerned with human error. We are not stopping anyone from running that CI twice, right?